assets/js/state/sagas/databases.test.js

```diff
@@ -82,4 +102,52 @@ describe('SAP Systems sagas', () => {
       }),
     ]);
   });
+
+  it('should deregister the database instance', async () => {
+    const instance = databaseInstanceFactory.build();
+    const { sap_system_id, host_id, instance_number } = instance;
+
+    axiosMock
+      .onDelete(
+        `/databases/${sap_system_id}/hosts/${host_id}/instances/${instance_number}`
+      )
+      .reply(204, {});
+
+    const dispatched = await recordSaga(deregisterDatabaseInstance, {
+      payload: instance,
+    });
+
+    expect(dispatched).toEqual([
+      setDatabaseInstanceDeregistering(instance),
+      setDatabaseInstanceDeregisteringToSAPSystem(instance),
+      setDatabaseInstanceNotDeregistering(instance),
+      setDatabaseInstanceNotDeregisteringToSAPSystem(instance),
+    ]);
+  });
+
+  it('should notify error on database instance deregistration request', async () => {
```

Comment:
I think I'm missing something in the title that tells me why it should notify an error, something like:
`should notify error if 404 is received during database instance deregistration request`

wdyt?

Reply:
What do you think of?
`should notify an error on database instance deregistration request failure`.
Being 404 is just a test coincidence. It would notify the same error in other cases (422 for example)

Reply:
Perfect :+1: